PlaywrigthUITests/PageObjects/DemoQAWebTablesPage.cs

```diff
@@ -87,18 +93,124 @@ public async Task VerifyTableRowContent(string headerName = "First Name", string
         }
     }
 
-    public async Task VerifyPopupVisible()
+    public async Task<bool> VerifyPopupVisible(bool result)
```

Comment:
since no assertion here anymore and it returns value, name should be IsPopupVisible

Reply:
result is not used

Reply:
fixed

Reply:
changes reverted